openmeter/productcatalog/plan/service.go

```diff
@@ -328,242 +322,249 @@ func (i ArchivePlanInput) Validate() error {
 	return nil
 }
 
-type NextPlanInput struct {
-	// NamespacedID
-	models.NamespacedID
+// type NextPlanInput struct {
```

Comment:
why commented out?

Reply:
The functionality provided by `NextPlan` API is also available via `CreatePlan` API. The only difference is that the former allows creating a new version with optinal plan payload while the latter requires a plan to be sent as part of the request. But I tend to agree that it does not worth to keep/maintain 2 API endpoints for the same operation.